Khan/perseus
# [Locked Figure Labels] Add/edit/delete locked vector labels

## Summary:
Add the ability to add/edit/delete labels for locked vectors
within the interactive graph editor.

Issue: https://khanacademy.atlassian.net/browse/LEMS-2348

## Test plan:
- Go to http://localhost:6006/?path=/story/perseuseditor-widgets-interactive-graph--mafs-with-locked-vector-labels-flag
- Go to locked figures
- Open the locked vector settings
- Confirm that the label settings are there
- Play around with the label settings and confirm that it moves
  and updates with the vector

<img width="335" alt="image" src="https://github.com/user-attachments/assets/78e0bba8-981f-46c4-8158-6df89517c53e">
